scripts/deploy.js

```diff
@@ -126,11 +126,11 @@ async function main() {
     const vault = await Vault.deploy(proxy.address);
     await vault.deployed();
 
-    const token = await Token.deploy("Wrapper Namada", "WNAM", [tokenSupply], [vault.address]);
-    await token.deployed();
+    const wnamToken = await Token.deploy("Wrapper Namada", "WNAM", [wnamTokenSupply], [vault.address]);
+    await wnamToken.deployed();
 
-    const tokenWhitelist = tokenWhitelistPrompt.length == 0 ? [token.address] : tokenWhitelistPrompt.split(',').map(token => token.trim()).concat(token.address)
-    const tokenCaps = tokenCapsPrompt.length == 0 ? [] : tokenCapsPrompt.split(',').map(cap => parseInt(cap))
+    const tokenWhitelist = tokenWhitelistPrompt.length == 0 ? [wnamToken.address] : tokenWhitelistPrompt.split(',').map(token => token.trim()).concat(wnamToken.address)
+    const tokenCaps = tokenCapsPrompt.length == 0 ? [wnamTokenSupply] : tokenCapsPrompt.split(',').map(cap => parseInt(cap)).concat(wnamTokenSupply)
```

Comment:
Here is where we concatenate the `wnamTokenSupply` similar to how we concatenate `wnamToken.address`